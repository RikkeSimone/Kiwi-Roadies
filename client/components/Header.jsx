import React from 'react'

function Header () {
  return (
    <div className="header">
      <img className="logo" src="./graphics/kiwiroadieslogo.png"/>
    </div>
  )
}

export default Header
