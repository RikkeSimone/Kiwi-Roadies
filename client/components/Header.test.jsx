import React from 'react'
import { renderWithRedux } from '../testing/utils'
import '@testing-library/jest-dom'
import 'regenerator-runtime/runtime'

import Header from './Header'

test('Header contains a logo', async () => {
  const { asFragment } = renderWithRedux(<Header />)
  expect(asFragment()).toMatchSnapshot()
})
