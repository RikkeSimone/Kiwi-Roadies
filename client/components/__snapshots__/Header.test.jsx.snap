// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header contains a logo 1`] = `
<DocumentFragment>
  <div
    class="header"
  >
    <nav
      role="navigation"
    >
      <div
        id="menuToggle"
      >
        <input
          type="checkbox"
        />
        <span />
        <span />
        <span />
        <ul
          id="menu"
        >
          <p>
            Made with love by Ariana, Rikke, Kim and Jonty
          </p>
        </ul>
      </div>
    </nav>
    <img
      class="logo"
      src="./graphics/kiwiroadieslogo-no-road.png"
    />
    <img
      class="github-gato"
      src="./github-gato.png"
    />
  </div>
</DocumentFragment>
`;
